### PR TITLE
Modify rule S4275: Adapt to LaYC

### PR DESCRIPTION
**Pull request by ilia-kebets-sonarsource**

### CODE REVIEW

File: rules/S4275/javascript/rule.adoc [hunk 1]
```diff
@@ -2,9 +2,17 @@
 
 include::../description.adoc[]
 
-=== Noncompliant code example
+== How to fix it
 
-[source,javascript]
+To Fix this issue, return the corresponding property in getters and assign the parameter value to the corresponding property in setters.
+
+Underscore prefixes for fields are supported, so `setY()` can assign a value to `_x`.
```
Comment (vilchik-elena) [inline rules/S4275/javascript/rule.adoc:9]:
```
Referencing property with an underscore (e.g. `this._x`) also considered compliant
```

Comment (vilchik-elena) [inline rules/S4275/javascript/rule.adoc:9]: I would move this paragraph to the before the next pair of examples

**Review by vilchik-elena — CHANGES_REQUESTED** [review]

File: rules/S4275/javascript/rule.adoc [hunk 2]
```diff
@@ -66,15 +58,51 @@ class A {
...
   _y: 0,
   get x() {
     return this._x;
   }
 };
+----
+
+==== Noncompliant code example
+
+[source,javascript,diff-id=3,diff-type=noncompliant]
```
[the remark this answers is not in the document]
Comment (ilia-kebets-sonarsource) [reply rules/S4275/javascript/rule.adoc:91]: yep

File: rules/S4275/javascript/rule.adoc [hunk 3]
```diff
@@ -2,9 +2,17 @@
 
 include::../description.adoc[]
```
[the remark this answers is not in the document]
Comment (ilia-kebets-sonarsource) [reply rules/S4275/javascript/rule.adoc:3]: it is for private fields like `#x`, but it is not the main case.

File: rules/S4275/javascript/rule.adoc (hunk 1 above)
Comment (ilia-kebets-sonarsource) [reply rules/S4275/javascript/rule.adoc:9]: I don't understand

File: rules/S4275/javascript/rule.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (ilia-kebets-sonarsource) [reply rules/S4275/javascript/rule.adoc:3]: done

File: rules/S4275/javascript/rule.adoc [hunk 4]
```diff
@@ -2,9 +2,17 @@
...
 
-=== Noncompliant code example
+== How to fix it
 
-[source,javascript]
+To Fix this issue, return the corresponding property in getters and assign the parameter value to the corresponding property in setters.
```
[the remark this answers is not in the document]
Comment (ilia-kebets-sonarsource) [reply rules/S4275/javascript/rule.adoc:13]: It's a bit heavy, but it's not ambiguous

File: rules/S4275/javascript/rule.adoc (hunk 1 above)
Comment (vilchik-elena) [reply rules/S4275/javascript/rule.adoc]: I tried to phrase it a bit differently

Comment (ilia-kebets-sonarsource) [reply rules/S4275/javascript/rule.adoc]: done

**Pull request closed without merging**